sortfilterproxymodel.qbs

```diff
@@ -1,56 +1,62 @@
-import qbs 1.0;
+import qbs 1.0
 
 Product
 {
-    name: "sortfilterproxymodel";
+    name: "sortfilterproxymodel"
     type: "dynamiclibrary"
-    targetName: "sortfilterproxymodel";
-    destinationDirectory: project.deployDirectory;
+    targetName: "sortfilterproxymodel"
 
-    readonly property stringList qtModules : ["core", "qml", "gui"];
+    destinationDirectory: project.deployDirectory
+
+    readonly property stringList qtModules:
+    [
+        "core",
+        "qml",
+        "gui"
+    ]
 
     cpp.defines:
     {
-        var defines = [];
+        var defines = []
         
-        defines.push('QT_NO_KEYWORDS');
-        defines.push('QRMVISUALCONFIG_LIBRARY_BUILD');
+        defines.push('QT_NO_KEYWORDS')
+        defines.push('SORT_FILTER_PROXY_MODEL_LIBRARY_BUILD')
 
         if (qbs.buildVariant === "release")
         {
-            defines.push('QT_NO_DEBUG_OUTPUT=1');
+            defines.push('QT_NO_DEBUG_OUTPUT=1')
         }
         
-        return defines;
+        return defines
     }
 
-    cpp.includePaths: ["."];
+    cpp.includePaths: [ "." ]
```

Comment:
We don't have any set rules for this, I was just trying to keep things as consistent as possible with the most recent Qbs files we've been writing. Most of the style "decisions" there have been made by @crozeg, but they're not really rules. Just aiming for consistency.